.tekton/run-acceptance-test.yaml

```diff
@@ -0,0 +1,146 @@
+    pipelinesascode.tekton.dev/task-37: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-pipelines-tests.yaml"
+    pipelinesascode.tekton.dev/task-38: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mini-must-gather.yaml"					
+    pipelinesascode.tekton.dev/task-39: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-triggers-tests.yaml"
+    pipelinesascode.tekton.dev/task-40: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mirror-and-tag-imagestreams.yaml"			
+    pipelinesascode.tekton.dev/task-41: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/verify-binaries-and-sha256sum.yaml"
+    pipelinesascode.tekton.dev/task-42: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mirror-operator-images.yaml"				
+    pipelinesascode.tekton.dev/task-43: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/verify-binaries-on-mac.yaml"
+    pipelinesascode.tekton.dev/task-44: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/release-tests.yaml"					
+    pipelinesascode.tekton.dev/task-45: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/whitelist-domains.yaml"
+    pipelinesascode.tekton.dev/task-46: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/run-tfa-classifire.yaml"
+    pipelinesascode.tekton.dev/max-keep-runs: "5"  
+spec:
+  params:
+    - name: repo_url
+      value: "{{ repo_url }}"
+    - name: revision
+      value: "{{ revision }}"
+  pipelineSpec:
+    params:
+      - name: repo_url
+      - name: revision
+    workspaces:
+      - name: source
+      - name: basic-auth
+      - name: cache
+    tasks:
+      - name: fetch-repository
+        taskRef:
+          kind: ClusterTask
+          name: git-clone
+        workspaces:
+          - name: output
+            workspace: source
+          - name: basic-auth
+            workspace: basic-auth
+        params:
+          - name: url
+            value: $(params.repo_url)
+          - name: revision
+            value: $(params.revision)
+        taskSpec:
+          pipelineRef:
+            name: acceptance-tests
+          params:
+            - name: ARCH
+              value: "linux/amd64"
+            - name: CATALOG_SOURCE
+              value: custom-operators
+            - name: CLUSTER_POOL
+              value: "openstack"
+            - name: CLI_TESTS_TYPE
+              value: downstream
+            - name: CLUSTER_NAME
+              value: hive
+            - name: INDEX
+              value: "766649"
+            - name: INSTALLER
+              value: "hive"
+            - name: INSTALL_PIPELINES_OPERATOR
+              value: "true"
+            - name: INSTALL_SERVERLESS_OPERATOR
+              value: "true"
+            - name: OPERATOR_ENVIRONMENT
+              value: pre-stage
+            - name: OPERATOR_VERSION
+              value: "1.15.1"
```

Comment:
This would be `5.0` for master branch, on other branches we would probably need to update this manually 🤷🏻